lib/cfnguardian/resources/dms_task.rb

```diff
@@ -0,0 +1,42 @@
+module CfnGuardian::Resource
+  class DMSTask < Base
+    
+    def default_alarms             
+      alarm = CfnGuardian::Models::DMSTaskAlarm.new(@resource)
+      alarm.name = 'CDCLatencySourceCritical'
+      alarm.metric_name = 'CDCLatencySource'
+      alarm.statistic = 'Minimum'
+      alarm.threshold = 30
+      alarm.evaluation_periods = 10
+      alarm.enabled = false
+      @alarms.push(alarm)
+      
+      alarm = CfnGuardian::Models::DMSTaskAlarm.new(@resource)
+      alarm.name = 'CDCLatencySourceWarn'
+      alarm.metric_name = 'CDCLatencySource'
+      alarm.statistic = 'Minimum'
+      alarm.threshold = 30
+      alarm.evaluation_periods = 1
+      alarm.enabled = false
+      @alarms.push(alarm)
+
+      alarm = CfnGuardian::Models::DMSTaskAlarm.new(@resource)
+      alarm.name = 'CDCLatencyTargetCritical'
+      alarm.metric_name = 'CDCLatencyTarget'
+      alarm.statistic = 'Minimum'
+      alarm.threshold = 30
+      alarm.evaluation_periods = 10
+      alarm.enabled = false
+      @alarms.push(alarm)
+
+      alarm = CfnGuardian::Models::DMSTaskAlarm.new(@resource)
```

Comment:
over-ride required for alarm action to warning?

Reply:
Same as above